docs/DeveloperGuide.md

```diff
@@ -547,45 +624,65 @@ testers are expected to do more *exploratory* testing.
 1. Editing a Session while all Sessions are being shown
 
    1. Prerequisites: Multiple Sessions in the list can be viewed on the right panel of the GUI.
-
-   1. Test case: `sedit 1 g/Machoman`<br>
+1. Test case: `sedit 1 g/Machoman`<br>
       Expected: First Session's gym location is edited.
       Details of the edited session is shown in the status message. Timestamp in the status bar is updated.
-
    1. Test case: `sedit 1 at/29/09/2020 1600 t/120`<br>
-      Expected: First Session timing is edited.
+   Expected: First Session timing is edited.
       Details of the edited session is shown in the status message. Timestamp in the status bar is updated.
-
    1. Other incorrect edit commands to try: `sedit`, `sedit x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
+### Adding a Schedule
+
+1. Adding a Schedule while all Clients and Sessions are being shown
+
+   1. Prerequisites: Multiple Clients and Sessions in the list can be viewed on the left and right panel of the GUI respectively.
+   1. Test case: `schadd c/1 s/1`<br>
+      Expected: Add a Schedule associated with Client of index 1 in the Client List and Session of index 1 in the Session List.
+      Details of the added Schedule is shown in the status message. Timestamp in the status bar is updated.
+   1. Test case: `schadd s/1 c/2`<br>
+      Expected: Add a Schedule associated with Client of index 2 in the Client List and Session of index 1 in the Session List.
+      Details of the added Schedule is shown in the status message. Timestamp in the status bar is updated.
+   1. Other incorrect Add Schedule commands to try: `schadd c/1`, `schadd c/0 s/2`, `schadd c/x s/y`, `...` (where x is larger than the Client List size or y is larger than the Session List size)<br>
+      Expected: No Schedule is added. Error details shown in the status message.
+
 ### Editing a Schedule
 
 1. Editing a Schedule while all Schedules are being shown
 
    1. Prerequisites: Multiple Schedules in the list can be viewed on the main panel of the GUI.
-
-   1. Test case: `editschedule c/1 s/1 us/2`<br>
+1. Test case: `schedit c/1 s/1 us/2`<br>
       Expected: Edit Schedule with client index 1 and session index 1 is edited to session index 2.
       Details of the edited schedule is shown in the status message. Timestamp in the status bar is updated.
```

Comment:
Can remove the timestamp sentences in this section too

Reply:
i think maguire is editing this part